app/controllers/users/products_controller.rb

```diff
@@ -14,7 +14,11 @@ def set_user
   end
 
   def set_products
-    @products = user.products.list.order_for_list
+    @products = if params[:target] == 'self_assigned'
+                  user.products.where(checker_id: current_user.id).list.order_for_list
```

Comment:
Productモデルに同様のSQLを投げるscopeがあるので、scopeを使った書き方にすると共通化できて良さそうですが、いかがでしょうか

```suggestion
                  user.products.self_assigned_product(current_user.id).list.order_for_list
```
https://github.com/fjordllc/bootcamp/blob/bd2cac8ef23021278f85550da22df81516a7e9c9/app/models/product.rb#L39

Reply:
おっしゃるとおりscopeを使いまわすことができますね😸✨
修正いたしました。
ご提案ありがとうございます！